src/dmd/func.d

```diff
@@ -210,6 +210,11 @@ extern (C++) class FuncDeclaration : Declaration
          */
         VarDeclaration vthis;
 
+        /**
+         * Is 'this' an pointer to a static array holding two contexts.
```

Comment:
Is 'this' _a_ pointer (yes English is stupid and inconsistent)

Reply:
done